pages/_watch/replay/_replay.vue

```diff
@@ -611,8 +613,13 @@
           ? data.timestamp.toDate()
           : null;
 
+        // Fix relative urls...
+        let szTempString = data.url;
+        // Probably we don't need to check it against https or http, simple check is enough
+        if(szTempString.length > g_iHttpStringLength && szTempString.startsWith(g_szHttp)) szTempString = `https://${szTempString}`;
```

Comment:
This can incorrectly cause a url that starts with `https://example.com` to be changed to `https://https://example.com`

This is because this checks for a string starting with just `http`, which may incorrectly trigger.

Reply:
What?

Reply:
assume `data.url = https://example.com/somevideo.mp4`

Then `szTempString.length > g_iHttpStringLength` is `33 > 4` (true)

AND

`szTempString.startsWith('http')` is also `true`, so 
```
szTempString = `https://${szTempString}`
// will become...
szTempString = `http://https://example.com/somevideo.mp4`
```

Reply:
```suggestion
        if(szTempString.length > g_iHttpStringLength && !szTempString.startsWith(g_szHttp)) szTempString = `https://${szTempString}`;
```

Reply:
It's not true dude.
```
js
startsWith == (a[0] === 'h' && a[1] === 't' && a[2] === 't' && a[3] === 'p')
```

Isn't kinda obvious thing?

Ok, now i see, missed `!`.